dist/fabric.js

```diff
@@ -7233,6 +7243,7 @@ fabric.ElementsParser = function(elements, callback, options, reviver, parsingOp
     */
     render: function(ctx, left, top, styleOverride, fabricObject) {
       styleOverride = styleOverride || {};
+      styleOverride.resizingStrokeColor = this.resizingStrokeColor;
```

Comment:
This seems a little circular - wouldn't we want styleOverride to override the base values if it is defined?

Reply:
Fixed! Thanks